app/models/binary/DataSet.scala

```diff
@@ -134,6 +136,12 @@ class DataSetDAO @Inject()(sqlClient: SQLClient,
       parsed <- Fox.combined(r.toList.map(parse))
     } yield parsed
 
+  def isEmpty(implicit ctx: DBAccessContext): Fox[Boolean] =
```

Comment:
previously queried all, returned data and then tested isEmpty in scala

Reply:
:+1: